Tetrahedral_remeshing/test/Tetrahedral_remeshing/test_tetrahedral_remeshing_of_one_subdomain.cpp

```diff
@@ -37,12 +39,13 @@ bool generate_input_two_subdomains(const std::size_t nbv, Remeshing_triangulatio
   }
   CGAL_assertion(tr.is_valid(true));
 
+#ifdef CGAL_TETRAHEDRAL_REMESHING_GENERATE_INPUT_FILES
   std::string filename("data/triangulation_two_subdomains.binary.cgal");
   std::ofstream os(filename, std::ios_base::out | std::ios_base::binary);
   os << "binary CGAL c3t3\n";
   CGAL::set_binary_mode(os);
-
-  return !!(os << tr);
+  os.close();
```

Comment:
Calling `close()` explicit is not really necessary at the end of a scope, because the steam is about to be destroyed (and the destructor calls `close()`, among other things).

Reply:
I understand, thank you!